plugins/techdocs/src/client.ts

```diff
@@ -97,7 +97,12 @@ export class TechDocsClient implements TechDocsApi {
       headers: token ? { Authorization: `Bearer ${token}` } : {},
     });
     const res = await request.json();
-
+    if (res.error) {
```

Comment:
Hm it's perhaps a bit unfortunate to rely on the body serialization format of the service here ... we can just use the raw http response code which is more straightforward. The most simple fix for now, without touching anything else, is to just immediately after the `fetch` do

```ts
if (request.status === 404) {
  return undefined;
}
```

and then leave the rest as-is for a future fix

Reply:
Updated